.buildkite/pipeline.yml

```diff
@@ -0,0 +1,11 @@
+steps:
+  - label: ":hammer: Build"
+    command:
+      - "npm install"
```

Comment:
1. Nit: you can omit the quotations here and on the line below.
2. Let's use `npm ci` here - it'll be much faster.

Reply:
revised.